composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
+        "satooshi/php-coveralls": "^0.7.1"
```

Comment:
@SpacePossum BTW, concerning your ref, this is unrelated.

Related issue: https://github.com/sebastianbergmann/phpunit/issues/2219